tests/type_inference/type_inference_visitor_unittest.py

```diff
@@ -0,0 +1,305 @@
+"""Unittest for the type_inference_visitor."""
+
+import unittest
+from astroid.transforms import TransformVisitor
+import astroid
+from type_inference_visitor import *
+
+type_visitor = TransformVisitor()
```

Comment:
@nigef has a nice use of `setUpClass` to set the TransformVisitor variable in #88. Please follow his example to do the same.